src/tests/system/tests/test_authentication.py

```diff
@@ -226,3 +231,142 @@ def test_authentication__user_login_when_the_provider_is_offline(
 
     assert client.auth.parametrize(method).password(user, correct), "User failed login!"
     assert not client.auth.parametrize(method).password(user, wrong), "User logged in with an incorrect password!"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.preferred_topology(KnownTopology.LDAP)
+@pytest.mark.parametrize(
+    "home_key",
+    ["user", "uid", "fqn", "domain", "first_char", "upn", "default", "lowercase", "substring", "literal%"],
+)
+@pytest.mark.importance("medium")
+def test_authentication__user_login_with_overriding_home_directory(
```

Comment:
Hi,

why is this test needed here? Imo only `test_authentication__user_login_with_modified_PAM_stack_provider_is_offline()` is needed in the backport and the include form line 9. All the other changes can be removed from the backport.

bye,
Sumit